src/data/form/Leads.js

```diff
@@ -8,7 +8,11 @@ export const FIELDS = {
     input: "description",
     width: 12,
     texts: [
-      `Welcome to ${data.name}. Thank you for considering to become a lead, we appreciate your enthusiasm to join ${data.name}. ${data.name} is a ${data.description} hackathon spanning ${data.length} on ${data.date}.`,
+      `Welcome to ${data.name} ${
+        parseInt(data.year) + 1
```

Comment:
this should be based on the date attribute

Reply:
Is the the date attribute supposed to be a string or a date? If it's supposed to be a string, how would I get/change only the year part? If it's supposed to be a date, can I change the value to a random date?